smart-contracts/src/Challenge_7.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >=0.6.0 <0.9.0;
+
+import '@openzeppelin/contracts/token/ERC721/ERC721.sol';
+
+/// @title Challenge 7
+/// @author Byont Labs
+/// @notice A barebones NFT contract example with overkill documentation for learning purposes
+// contract Contract is ERC721 {
+//   /// @dev Library that can convert uint256 to string. (by the way, uint is short for uint256)
+//   using Strings for uint256;
+
+//   /// @dev Constructor arguments are passed to the contract on contract creation, see Challenge_5.t.sol
+//   constructor(
```

Comment:
Its really sub-ideal to review code thats all commented out, I wonder why you did that